flow-typed/npm/downshift_v2.x.x.js.flow

```diff
@@ -114,27 +114,97 @@ declare module downshift {
   declare export type StateChangeFunction<Item> = (
     state: DownshiftState<Item>,
   ) => StateChangeOptions<Item>
+
+  declare export type GetRootPropsReturn = {
+    role: 'combobox';
+    // 'aria-expanded': boolean;
+    // 'aria-haspopup': 'listbox';
+    // 'aria-owns': string | null;
+    // 'aria-labelledby': string;
+  }
   declare export interface GetRootPropsOptions {
     refKey: string;
   }
-  declare export interface GetInputPropsOptions
-    extends React.HTMLProps<HTMLInputElement> {}
-  declare export interface GetLabelPropsOptions
-    extends React.HTMLProps<HTMLLabelElement> {}
+
+  declare type GetToggleButtonCallbacks = {
+    onMouseMove: (e: React.MouseEvent<Element>) => void;
+    onMouseDown: (e: React.MouseEvent<Element>) => void;
+    onBlur: (e: React.MouseEvent<Element>) => void;
```

Comment:
This is probably not `MouseEvent`

Reply:
Hmmm... I'm confused. If it's not `MouseEvent` then why is it typed to be a `MouseEvent` here?

Reply:
I meant: this is an error I'm aware of

Reply:
Can this not just be typed with a `SyntheticEvent`?